packages/fast-components/src/design-system-provider/design-system-provider.template.ts

```diff
@@ -1,6 +1,144 @@
 import { html } from "@microsoft/fast-element";
 import { DesignSystemProvider } from "./design-system-provider";
+import {
+    accentFill,
+    accentFillLargeRest,
+    accentFillRest,
+    accentForeground,
+    accentForegroundCut,
+    accentForegroundCutLarge,
+    accentForegroundLargeRest,
+    accentForegroundRest,
+    DesignSystemResolver,
+    neutralFillCard,
+    neutralFillInputRest,
+    neutralFillRest,
+    neutralFillStealthRest,
+    neutralFillToggleRest,
+    neutralFocus,
+    neutralForegroundHint,
+    neutralForegroundHintLarge,
+    neutralForeground,
+    neutralForegroundToggle,
+    neutralForegroundToggleLarge,
+    neutralLayerCard,
+    neutralLayerCardContainer,
+    neutralLayerFloating,
+    neutralLayerL1,
+    neutralLayerL1Alt,
+    neutralLayerL2,
+    neutralLayerL3,
+    neutralLayerL4,
+    neutralOutlineRest,
+    accentForegroundLarge,
+    neutralFill,
+    neutralFillStealth,
+    neutralFillToggle,
+    neutralFillInput,
+    accentFillLarge,
+    neutralOutline,
+    neutralDividerRest,
+    neutralFocusInnerAccent,
+} from "@microsoft/fast-components-styles-msft";
 
 export const DesignSystemProviderTemplate = html<DesignSystemProvider>`
+  <template style="

```

Comment:
It is pretty cool that we can do this. It's probably not great perf-wise...but who knows? In my perf work recently, I've often been surprised by what I thought would be slow and ultimately didn't end up mattering.

Reply:
yep! It is super useful. This will likely have a performance hit just because it's generating *every* color regardless of use and not doing any property inheritance, but it will work well enough for the time being.